selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -353,9 +367,22 @@ public void waitItemIsNotPresentVisibleArea(String item) {
    */
   public void selectItem(String path) {
     String locator = "//div[@path='/" + path + "']/div";
+
     waitItem(path);
-    WebElement item = seleniumWebDriver.findElement(By.xpath(locator));
-    item.click();
+    try {
+      new WebDriverWait(seleniumWebDriver, EXPECTED_MESS_IN_CONSOLE_SEC)
+          .until(ExpectedConditions.visibilityOfElementLocated(By.xpath(locator)))
+          .click();
+    } catch (StaleElementReferenceException ex) {
```

Comment:
Such cases should be commented with short explanation of workaround, as a rule, to explain why we can ignore an exception and can try to repeat operation again.